File: Semestrul 4/MPP2/Problema8SC_CSharp/Networking/dto/EnrollmentDTO.cs
namespace Networking.dto;

[Serializable]
public class EnrollmentDTO
{
    public long ChildId { get; set; }
    public long EventId { get; set; }

    public EnrollmentDTO(long childId, long eventId)
    {
        ChildId = childId;
        EventId = eventId;
    }

    public override string ToString()
    {
        return $"EnrollmentDTO[{ChildId} {EventId}]";
    }
}

File: Semestrul 4/MPP2/Problema8SC/Problema8SC/Problema8Networking/src/main/java/problema8/network/dto/EnrollmentDTO.java
package problema8.network.dto;

import problema8.model.Child;
import problema8.model.Enrollment;
import problema8.model.Event;

import java.io.Serializable;

public class EnrollmentDTO implements Serializable {
    private Long childId;
    private Long eventId;

    public EnrollmentDTO(Long childId, Long eventId) {
        this.childId = childId;
        this.eventId = eventId;
    }

    public Long getChildId() {
        return childId;
    }

    public void setChildId(Long childId) {
        this.childId = childId;
    }

    @Override
    public String toString() {
        return "EnrollmentDTO[" + childId + " " + eventId + "]";
    }
}
